bin/postgres-ha/bootstrap/pre-bootstrap.sh

```diff
@@ -256,6 +262,8 @@ build_bootstrap_config_file() {
     then
         echo_info "Applying base bootstrap config to postgres-ha configuration"
         /opt/cpm/bin/yq m -i -x "${bootstrap_file}" "/opt/cpm/conf/postgres-ha-bootstrap.yaml"
+        # set the configured bootstrap method (e.g. initdb or pgbackrest)
+        sed -i "s/PGHA_BOOTSTRAP_METHOD/$PGHA_BOOTSTRAP_METHOD/g" "${bootstrap_file}"
```

Comment:
The `sed` approach aligns with how we do other similar replacements in the conf files, so was just staying consistent.

Is there a reason we don't want to use `sed`?

Reply:
Yeah, we’ve been trying to move towards ‘envsubst’

Reply:
Ah OK, had been thinking of that in regard to `expenv`, but no necessarily for `sed`.

I'll get the ball rolling with this script, and will update this one to use `expenv` instead.

Reply:
Happy to settle with sed if the rest of the file currently uses it.

Reply:
Does feel like `sed` is a bit cleaner for inline file replacements like this (vs. piping to other command, like we do in other places we use `expenv`, e.g. for `psql` or `kubectl`).  Otherwise, I'm looking at using multiple commands (e.g. `tee` in this example):

```bash
envsubst < postgres-ha-bootstrap.yaml | tee postgres-ha-bootstrap.yaml
```

Or the other alternative is creating a temporary file:.  

```bash
envsubst < postgres-ha-bootstrap.yaml > postgres-ha-bootstrap.yaml.tmp
mv postgres-ha-bootstrap.yaml.tmp postgres-ha-bootstrap.yaml
```

But both of these make the `sed` single tool, single-line approach attractive.  Def open to any other suggestions, but at this point leaning towards keeping `sed` for this sort of thing.

Reply:
Per the above, sticking with `sed` for now.